.github/workflows/onPRClose.yml

```diff
@@ -3,41 +3,77 @@ on:
   pull_request:
     types: [closed]
 jobs:
-  extractIssueNumber:
-    name: Extract Issue Number
+
+  get-pr-number:
```

Comment:
@keifgwinn  why do you need this? 

Unless i'm mistaken as this yml is only ran on PR close, `github.event.number` should always get you the PR number?